tests/EntityExistValidatorTest.php

```diff
@@ -158,4 +160,41 @@ public function testValidateInvalidEntity(): void
 
         $this->validator->validate(1, $constraint);
     }
+
+    /**
+     * @requires PHP 8
+     */
+    public function testValidateFromAttribute()
+    {
+        $this->context->expects($this->never())->method('buildViolation');
+
+        $classMetadata = new ClassMetadata(EntityDummy::class);
+        (new AnnotationLoader())->loadClassMetadata($classMetadata);
```

Comment:
@Nyholm the AnnotationLoader class expect a Reader argument for Symfony version < 5.2. That's why the test is failling.

Reply:
Interesting. 
Is there a workaround or should we drop SF <5.2?